src/main/java/uk/gov/digital/ho/hocs/casework/api/SomuItemService.java

```diff
@@ -45,31 +45,37 @@ public SomuItem getSomuItem(UUID caseUuid, UUID somuTypeUuid) {
             return somuItem;
         }
         else {
-            throw new ApplicationExceptions.EntityNotFoundException(String.format(NOT_FOUND_MESSAGE, somuTypeUuid), SOMU_ITEM_NOT_FOUND);
+            log.info("Could not find somu item for case {} for somu type {}, creating one. Event {}", caseUuid, somuTypeUuid, value(EVENT, SOMU_ITEM_RETRIEVED));
+            return addSomuItem(caseUuid, somuTypeUuid, null);
         }
     }
 
-    public SomuItem upsertSomuItem(UUID caseUUID, UUID somuTypeuuid, String data) {
-        log.debug("Upserting Somu Item of Type: {} for Case: {}", somuTypeuuid, caseUUID);
+    public SomuItem upsertSomuItem(UUID caseUuid, UUID somuTypeUuid, String data) {
```

Comment:
I appreciate your effort to standardise the capitalisation here, but I think "UUID" is more common.

Reply:
As agreed, this will form part of the next PR in the series.